src/main/java/main/movie/model/ShoppingCart.java

```diff
@@ -0,0 +1,58 @@
+package main.movie.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
we don't need to generate the id for shopping cart - it should be the same as for user
use `@MapsId` to have only one `id` column in `shopping_cart` table